README.adoc

```diff
@@ -217,35 +239,30 @@ If your build runs successfully, you'll see an output similar to the following:
 
 [role="no_copy"]
 ----
-Sending build context to Docker daemon  148.9MB
-Step 1/4 : FROM openliberty/open-liberty:kernel-java8-openj9-ubi
-latest: Pulling from library/openliberty/open-liberty:kernel-java8-openj9-ubi
-7b8b6451c85f: Pull complete 
-ab4d1096d9ba: Pull complete 
-e6797d1788ac: Pull complete 
-e25c5c290bde: Pull complete 
-27b2fbbc72b1: Pull complete 
-aa35dfd74487: Pull complete 
-1ade56d64661: Pull complete 
-3493ef172f60: Pull complete 
-cd6dff6d245d: Pull complete 
-82450af30afc: Pull complete 
-Digest: sha256:b7576e4278030537765d4185c4641ee2769194226263f979109ed4fa0e1aa4e4
-Status: Downloaded newer image for openliberty/open-liberty:kernel-java8-openj9-ubi
- ---> 45d4b67ace5b
-Step 2/4 : RUN ln -s /opt/ol/wlp/usr/servers /servers
- ---> Running in cbeb275770ab
-Removing intermediate container cbeb275770ab
- ---> 937183f8460b
-Step 3/4 : ENTRYPOINT ["/opt/ol/wlp/bin/server", "run"]
- ---> Running in 856a4bdec82b
-Removing intermediate container 856a4bdec82b
- ---> 6cf732381877
-Step 4/4 : CMD ["defaultServer"]
- ---> Running in 1a543a9e37d8
-Removing intermediate container 1a543a9e37d8
- ---> 8fdcad065d25
-Successfully built 8fdcad065d25
+Sending build context to Docker daemon  316.5MB
+Step 1/6 : FROM open-liberty
```

Comment:
@DannyS03 I saw `Step 1/6 : FROM openliberty/open-liberty:kernel-java8-openj9-ubi`

Reply:
Re-ran and successfully got the kernel image too.